components/contactus/contact.js

```diff
@@ -41,6 +44,24 @@ const Form = () => {
         
 </div>
 </div>
+
+<h1 className='text-center text-2xl font-bold text-black mt-5 p-4 '>OUR LOCATIONS</h1>
+<div className='flex flex-col  md:flex-row w-full items-center justify-center h-full mx-auto  gap-4 p-5 px-6'>
+   
+{/*  */}
+{Country.map((item,index)=>{
```

Comment:
please follow standard conventaions. use camelCasing for data. Country -> country